develop/schemadoc/Protocol/Protocol.Params.Param.ArrayOptions-options.md

```diff
@@ -261,3 +261,6 @@ directView=3301;onlyFilteredDirectView;filterChange=11601-9001,11602-9002,11603-
 ### volatile
 
 If you use this option, the primary keys will not be saved.
+
+> [!CAUTION]
```

Comment:
Also add this note to ColumnOptionOptionsOverview

Reply:
Done! Could you confirm if this is OK to merge now?